README.md

```diff
@@ -571,7 +595,7 @@ To estimate the emission and transition matrix, we normalise their rows so they
 ```python
 def normalise(matrix: ArrayLike) -> ArrayLike:
     """Normalise a matrix so each row sums to one."""
-    for i, row in enumerate(matrix):
+    for i, row in enumerate(matrix):                   # how does it know what row should be?
```

Comment:
It knows what `row` is because the `matrix` structure will behave like an iterator over rows when you use it as such. So if you do a for loop over a matrix

```python
for x in matrix:
   ...
```

you get the rows (i.e. `x` will iterate over rows). The `enumerate` function just gives us the number of the row as well.